syntax = "proto2";
option cc_generic_services = true;
package afa;

message ProtoRequest
{
    required string message_name = 1;
}

message ProtoResponse
{
    required string message_name = 1;
}

service RpcService
{
    rpc GetService (ProtoRequest) returns (ProtoResponse);
}